integration/openshift/lib.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Contain helper functions and variables for the OpenShift integration
+# test scripts.
+#
+set -e
+
+# ATTENTION: the caller of this lib should have the tests_dir exported
+#            correctly
+if [ ! -d "${tests_dir}" ]; then
+	echo >&2 "ERROR: Unable to find the Kata Containers tests directory"
+	exit 1
+fi
+
+source ${tests_dir}/.ci/lib.sh
+
+# The name of the Kubernetes runtimeClass resource for Kata Containers.
+#
+kata_runtimeclass=${kata_runtimeclass:-kata}
+
+# Build the openshift-tests binary.
+#
+# Return:
+#  The absolute path to the file.
+#
+function build_openshift_tests()
+{
+	local openshift_repo="github.com/openshift/origin"
+	local repo_dir="${GOPATH}/src/${openshift_repo}"
+	local cmd=""
+	{
+	[ -d "$repo_dir" ] || go get -d $openshift_repo
+	pushd "$repo_dir"
+		git checkout "release-$(get_version \
+			"externals.openshift.meta.newest-version")"
+		make WHAT=cmd/openshift-tests
+		cmd="$(realpath \
+			$(find -type f -executable -name "openshift-tests"))"
+	popd
+	} &>/dev/null
+	[ $? -eq 0 ] || return 1
```

Comment:
Is this correct? Return error and no string if the build is successful? But actually it isn't the build but the `popd` return value which is almost guaranteed to succeed isn't it (since we've already done a successful `pushd`)?

Reply:
The idea is to return the string or error if the build failed. But you are right, it is going to get the success of `popd`. I was almost sure I had tested this function...anyway, I will fix it by checking if `$cmd` is non-empty (or `return 1`).